### ISSUE TITLE
Setting coordinates in the publisher

### ISSUE THREAD

**Issue by svbergerem** [diaspora/diaspora#6440]
We now display locations on a map but users are not able to set the coordinates for a post via UI. This could be done by displaying a marker on a map which you can move to the desired place.

**Comment by kas70** [comment]
yep, this would be a nice feature. offering a search-function also would be a good thing, i think.
adding a location manually could be reused for the event-feature, too.
